pkg/settings/cresettings/settings.go

```diff
@@ -157,14 +155,8 @@ type Workflows struct {
 type cronTrigger struct {
 	RateLimit Setting[config.Rate]
 }
-type httpTriggerGlobal struct {
```

Comment:
removing httpTriggerGlobal and instead adding `Gateway*` limits to top-level `Schema` because the gateway limits not only affect httpTrigger but also Vault